src/internal/proc.js

```diff
@@ -244,6 +244,9 @@ export default function proc(
       iterator._deferredEnd && iterator._deferredEnd.resolve(result)
     } else {
       if(result instanceof Error) {
+        if(onerror) {
```

Comment:
I think we should put this inside the below `if(!task.cont) {` block. Because here it will notify all errors not just uncaught ones; for example

```js
function* child() {
  const {err, res} = yield call(fetch, 'url')
  if(err)
   throw new Error(err)
}

function* parent() {
  try {
    yield call(child)
  } catch(err) {
     // you don't want to report this one
  }
}
```